### ISSUE TITLE
Feature Request: Desktop Notificatiokns

### ISSUE THREAD

**Issue by Superluigi6** [kaypoz/reddit-notifier#8]
With Chrome's desktop notifications, you should have a feature that displays one whenever you get a message.

**Comment by bgrawi** [comment]
I agree, I can add this when I have some free time, or if you want to, submit a pull request!
Thanks :smile:!

**Comment by elamperti** [comment]
FYI, reddit is testing push notifications. If you go to _settings_ it will ask you for permission to show them.
